Source/Jira/_main_.env

```diff
@@ -3,6 +3,11 @@
 #JIRA_USER_AND_PASSWORD
 
 _t4dJiraVersion="7.13.0"
+if [[ "$_t4dJiraBaseURL" == "" ]] && [[ "$JIRA_SERVER_URL" != "" ]]; then
+    _t4dJiraBaseURL="${JIRA_SERVER_URL}"
+elif [[ "$JIRA_SERVER_URL" == "" ]] && [[ "$_t4dJiraBaseURL" != "" ]]; then
+    JIRA_SERVER_URL="${_t4dJiraBaseURL}"
+fi
```

Comment:
Wouldn't it be better with a "simple" `JIRA_SERVER_URL="${JIRA_SERVER_URL:_t4dJiraBaseURL}"` and keep a single var in jira code ?

Reply:
yes, but for now we need both behavior to be retrocompat as it's only a minor release. but yes in a perfect world i would remove _t4dJiraBaseURL from code